apis/core/v1alpha2/core_types.go

```diff
@@ -187,6 +187,11 @@ type ComponentParameter struct {
 
 // A ComponentSpec defines the desired state of a Component.
 type ComponentSpec struct {
+	// RevisionLimit is the maximum number of revisions that will
+	// be maintained. The default value is 10.
+	// +optional
+	RevisionLimit *int32 `json:"revisionLimit,omitempty"`
```

Comment:
I'm not sure if we need to make revisionLimit to be component instance level. Maybe system level is enough, that's mean it should be an argument of controller.

Reply:
👌

Reply:
The users are used to RevisionLimit in `Deployment` and `Statefulset`, is it good be an argument of controller?

Reply:
One of important meanings of OAM is to reduce the complexity of `Deployment` or `Statefulset`

Reply:
👌